acceptance/cli/cli.go

```diff
@@ -720,13 +720,30 @@ func matchSnapshot(ctx context.Context) error {
 		return err
 	}
 
-	stdout := snaps.MatchSnapshot(ctx, "stdout", status.stdout.String(), status.vars)
-	stderr := snaps.MatchSnapshot(ctx, "stderr", status.stderr.String(), status.vars)
+	var stdout, stderr error
 
+   formatJSON := true
```

Comment:
The issue is that not all output is in JSON format. Sometimes it's YAML. I think sometimes it might just be raw text (unclear). 

The first approach was to make it a parameter to the function, but I [suggested](https://github.com/enterprise-contract/ec-cli/pull/779#discussion_r1235649932) to make it dynamically determined :grimacing:  Otherwise, we'd have to introduce new matchers which seems a bit unnecessary - but definitely doable!